Examples/UseCases.ipynb

```diff
@@ -129,7 +144,16 @@
    "source": [
     "from pydkpro import DKProCoreTypeSystem as dts\n",
     "\n",
-    "cas.select(dts().token).as_text() "
+    "cas.select(dts().token).as_text() \n",
+    "\n",
+    "// REC: I wonder which version of the DKProCoreTypeSystem is used here...\n",
+    "\n",
+    "// REC: Why is dts() a method? It would seem quite redundant to always have to call this method.\n",
```

Comment:
Explaination is provided in this [comment](https://github.com/zesch/pydkpro/pull/14/commits/8b4f506bfe33aa06c1e1b2b69fc60edd70fb7325#r374156849)